### PR TITLE
Read-only labels filtering

### PR DESCRIPTION
**Pull request by m4dz**
This PR introduces the sidebar labels filtering

### CODE REVIEW

File: client/app/routes/index.coffee [hunk 1]
```diff
@@ -21,11 +22,18 @@ module.exports = class Router extends Backbone.Router
...
             contacts = new ContactsRouter 'contacts'
-        app.contacts.once 'sync', ->
-            app.layout.showContactsList()
-            app.layout.disableBusyState()
+            tags     = new TagsRouter 'tags'
+
+        @listenTo app.model, 'change:dialog', (model, id) ->
+            return if id
+            
+            path = if _ref = app.model.get 'filter' then "tags/#{_ref}"
```
Comment (frankrousseau) [inline client/app/routes/index.coffee]: Please avoid that kind of writing, it makes code reading hard.

**Comment by frankrousseau** [comment]
Can you add some comments to your code?

File: client/app/routes/index.coffee (hunk 1 above)
Comment (m4dz) [inline client/app/routes/index.coffee]: Do you prefer
```coffee
path = if (_ref = app.model.get 'filter') then "tags/#{_ref}"
```
or
```coffee
_ref = app.model.get 'filter'
path = if _ref then "tags/#{_ref}"
```
?

**Comment by m4dz** [comment]
> Can you add some comments to your code?

Anyway, I need to add comments to my whole codebase revamp :laughing:… I think it'll be my next PR, quickly.

**Comment by frankrousseau** [comment]
I prefer from far that one. Assignation in condition testing is ugly.

```coffeescript
_ref = app.model.get 'filter'
path = if _ref then "tags/#{_ref}"
```

**Pull request merged**